theme.inc

```diff
@@ -76,3 +77,32 @@ function _gent_base_get_entity_reference_file_data(&$variables, $field) {
 
   return $data;
 }
+
+/**
+ * Helper function to get the file data of a referenced File entity.
+ *
+ * @param mixed $item
+ *   The item to get the referenced entities from.
+ *
+ * @return array|null
+ *   The terms that are referenced.
+ *
+ * @throws \Drupal\Core\TypedData\Exception\MissingDataException
+ */
+function _gent_base_get_related_terrains($item) {
+  // Get the related terrain node.
+  $node = Node::load($item->id());
```

Comment:
Can we avoid doing a node load for each item?

Reply:
I don't think so... we need the referenced entity for each node...